charts/rancher-eks-operator/1.0.800/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-alpha1
```

Comment:
@PennyScissors should the former chart be maxing version to be compatible with your new system ?

Reply:
Yes, if we don't max the previous one then images from that chart will be added to the image list in future releases once my changes are in.